addons/dockable_container/samples/TestScene.gd

```diff
@@ -17,19 +17,19 @@ func _ready() -> void:
 		var checkbox = CheckBox.new()
 		checkbox.text = str(i)
 		checkbox.pressed = not _container.is_control_hidden(tabs[i])
```

Comment:
`BaseButton.pressed` got renamed to `button_pressed`.
Reference: https://docs.godotengine.org/en/latest/classes/class_basebutton.html#class-basebutton-property-button-pressed